website/screens/components/radio-group/specs/RadioGroupSpecsPage.tsx

```diff
@@ -0,0 +1,700 @@
+import DocFooter from "@/common/DocFooter";
+import Figure from "@/common/Figure";
+import HeadingLink from "@/common/HeadingLink";
+import Image from "@/common/Image";
+import Code from "../../../common/Code";
+import {
+  DxcList,
+  DxcStack,
+  DxcText,
+  DxcTable,
+  DxcLink,
+} from "@dxc-technology/halstack-react";
+import radioGroupBaseStates from "./images/radio_group_base_states.png";
+import radioGroupStates from "./images/radio_group_states.png";
+import radioGroupAnatomy from "./images/radio_group_anatomy.png";
+import radioGroupSpecs from "./images/radio_group_specs.png";
+
+const RadioGroupSpecsPage = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Specifications</HeadingLink>
+        <Figure caption="Component design specifications">
+          <Image src={radioGroupSpecs} alt="Component design specifications" />
+        </Figure>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>States</HeadingLink>
+        <DxcText as="p">
+          The following states are defined in the life cycle of the component:{" "}
+          <strong>unselected enabled</strong>, <strong>unselected hover</strong>
+          , <strong>unselected focus</strong>,{" "}
+          <strong>unselected disabled</strong>,{" "}
+          <strong>selected enabled</strong>, <strong>selected hover</strong>,{" "}
+          <strong>selected focus</strong> and <strong>selected disabled</strong>
+          .
+        </DxcText>
+        <Figure caption="Radio button states">
+          <Image src={radioGroupBaseStates} alt="Radio button input states" />
+        </Figure>
+        <DxcText as="p">
+          The following states are defined in the life cycle of the component:{" "}
+          <strong>enabled</strong>, <strong>error</strong>,{" "}
+          <strong>readonly</strong> and <strong>disabled</strong>.
+        </DxcText>
+        <Figure caption="Radio group states">
+          <Image src={radioGroupStates} alt="Radio group states" />
+        </Figure>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Anatomy</HeadingLink>
+        <Figure
+          caption={
+            <DxcList type="number">
+              <DxcText>Label</DxcText>
+              <DxcText>Helper text</DxcText>
+              <DxcText>Radio input</DxcText>
+              <DxcText>Radio input label</DxcText>
+              <DxcText>Error message</DxcText>
+            </DxcList>
+          }
+        >
+          <Image src={radioGroupAnatomy} alt="Radio button component anatomy" />
+        </Figure>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Design tokens</HeadingLink>
+        <Figure caption="Component design specifications">
+          <Image src={radioGroupSpecs} alt="Component design specifications" />
```

Comment:
The image `radioGroupSpecs` is duplicated, this one can be removed.

Reply:
Done!